vpr/src/base/read_place.cpp

```diff
@@ -306,10 +306,14 @@ void read_place_body(std::ifstream& placement_file,
  *
  * The architecture and netlist files used to generate this placement are recorded
  * in the file to avoid loading a placement with the wrong support file later.
+ *
+ * The is_initial_place bool indicates whether this is the initial placement or not. It is used to determine the
```

Comment:
remove is_initial_place parameter (re-factoring: this logic should be handled in the calling routine).